frontend/src/utils/YupSchema.ts

```diff
@@ -14,9 +14,13 @@ Yup.addMethod(Yup.string, 'optional', function optional() {
 });
 
 export const AccessRequestSchema = Yup.object().shape({
-  agency: Yup.number()
-    .min(1, 'Invalid Agency')
-    .required('Required'),
+  showAgency: Yup.boolean(),
+  agency: Yup.number().when('showAgency', {
+    is: true,
+    then: Yup.number()
+      .min(1, 'Invalid Agency')
+      .required('Required'),
+  }),
```

Comment:
Had to introduce a `showAgency` form field (hidden from the UI) to control whether `agency` is a required field. 

* MOTI: no agency (optional)
* CITZL agency is required